src/features/category/ui/routes/CategoryRoute/CategoryRoute.tsx

```
@@ -0,0 +1,102 @@
+  DrawerHeader,
+  DrawerTitle,
+  DrawerTrigger,
+} from '@/components/ui/drawer';
+import { Form, FormControl, FormField, FormItem } from '@/components/ui/form';
+import Layout from '@/shared/ui/layout/Layout';
+import { cn } from '@/shared/ui/styles/utils';
+import SubPageHeader from '@/shared/ui/SubPageHeader';
+import UnderlinedTextInput from '@/shared/ui/UnderlinedTextInput';
+
+const CategoryRoute: React.FC = () => {
+  const [input, setInput] = React.useState<string>('');
```

ESLint

[error] 'input' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)